sailpoint.sgnl.yaml

```diff
@@ -0,0 +1,1575 @@
+        type: String
+      - name: targetId
+        externalId: targetId
+        type: String
+      - name: targetType
+        externalId: targetType
+        type: String
+      - name: targetDisplayName
+        externalId: targetDisplayName
+        type: String
+
+      # Meta
+      - name: resourceType
+        externalId: $.meta.resourceType
+        type: String
+      - name: created
+        externalId: $.meta.created
+        type: DateTime
+      - name: lastModified
+        externalId: $.meta.lastModified
+        type: DateTime
+      - name: version
+        externalId: $.meta.version
+        type: String
+      - name: location
+        externalId: $.meta.location
+        type: String
+
+  # TODO: Identify unique and index attributes
+  AlertAttributes:
+    parent: Alert
+    displayName: SailpointAlertAttributes
+    externalId: attributes
+    description: Entity representing the attibutes associated with the Sailpoint Identity IQ Alert.
+    syncFrequency: HOURLY
+    syncMinInterval: 1
+    apiCallFrequency: SECONDLY
+    apiCallMinInterval: 1
+    attributes:
+      - name: key
+        externalId: key
+        type: String
+      - name: value
+        externalId: value
+        type: String
+
+  AlertActions:
+    parent: Alert
+    displayName: SailpointAlertActions
+    externalId: actions
+    description: Entity representing the actions associated with the Sailpoint Identity IQ Alert.
+    syncFrequency: HOURLY
+    syncMinInterval: 1
+    apiCallFrequency: SECONDLY
+    apiCallMinInterval: 1
+    attributes:
+      - name: type
+        externalId: type
+        type: String
+      - name: alertDefinitionName
+        externalId: alertDefinitionName
+        type: String
+      - name: resultId
+        externalId: result.id
+        type: String
+        indexed: true
+        uniqueId: true
+
+  AlertActionResult:
+    parent: actions
+    displayName: SailpointAlertActionResultNotifications
+    externalId: result
+    description: Entity representing the result associated with the Sailpoint Identity IQ Alert Action.
+    syncFrequency: HOURLY
+    syncMinInterval: 1
+    apiCallFrequency: SECONDLY
+    apiCallMinInterval: 1
+    attributes:
+      - name: name
+        externalId: name
+        type: String
+      - name: workflowName
+        externalId: workflowName
+        type: String
+        indexed: true
+        uniqueId: true
+
+  AlertActionResultNotifications:
+    parent: result
+    displayName: SailpointAlertActionResultNotifications
+    externalId: notifications
+    description: Entity representing the notifications associated with the Sailpoint Identity IQ Alert Action Result.
+    syncFrequency: HOURLY
+    syncMinInterval: 1
+    apiCallFrequency: SECONDLY
+    apiCallMinInterval: 1
+    attributes:
+      - name: name
+        externalId: name
+        type: String
+      - name: displayName
+        externalId: displayName
+        type: String
+      - name: emailAddresses
```

Comment:
Are we certain a user isn't able to register a email address twice (or a workflowName twice for  `AlertActionResult`), etc. If not, should we add a warning on these entity types as to what might break if multiple are specified?